test/e2e/page-objects/processes/login.process.ts

```diff
@@ -0,0 +1,30 @@
+import LoginPage from '../pages/login-page';
+import HomePage from '../pages/homepage';
+import { Driver } from '../../webdriver/driver';
+import { DEFAULT_GANACHE_ETH_BALANCE_DEC } from '../../constants';
+import { WALLET_PASSWORD } from '../../helpers';
+
+/**
+ * This method unlocks the wallet and verifies that the user lands on the homepage with the expected balance. It is designed to be the initial step in setting up a test environment.
+ *
+ * @param driver - The webdriver instance.
+ * @param password - The password used to unlock the wallet. Defaults to WALLET_PASSWORD.
+ * @param expectedBalance - The expected balance to be displayed on the homepage after successful login. Defaults to DEFAULT_GANACHE_ETH_BALANCE_DEC, reflecting common usage in test setups.
+ */
+export const loginWaitBalance = async (
```

Comment:
nit: could this naming be `loginWithBalanceValidaiton` since it's essentially the same?

Reply:
Hi, i think the plan is to migrate as much code as possible to TypeScript. The new method is written in TS and includes a default balance value and enhanced logging. It is also designed in line with the POM pattern, it aims to simplify the adoption of POM, ensuring all new tests fully follow the POM pattern, including the processes they call. We will permit both methods to exist, and promoting the use of the new TS POM method, until full migration is accomplished.

Reply:
oh yeah I'm okay with having both methods pre-existing. I was suggesting to make the name `loginWaitBalance` a bit more explicit like the other one, but it's also fine with me as it is :+1:

Reply:
oh i guess i misunderstood you ! sure, makes sense, i changed the name to `loginWithBalanceValidaiton` ! Thanks!